pkg/cmd/cli/cmd/rsh.go

```diff
@@ -143,5 +144,10 @@ func (o *RshOptions) Validate() error {
 
 // Run starts a remote shell session on the server
 func (o *RshOptions) Run() error {
+	// Insert the TERM into the command to be run
+	// TODO what if the container doesn't have `env`???
```

Comment:
I assume most usages of `oc rsh` use the default command (`/bin/sh`) -- "rsh" standing for remote shell. Minimal operating systems like Busybox, and Alpine include both `/bin/sh` and `/bin/env`.

For more control, one can always use `oc exec`.